ext/hal/nxp/mcux/devices/MK22F51212/fsl_device_registers.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * The Clear BSD License
```

Comment:
This is triggering the license failure in CI. NXP will soon release a new MCUXpresso SDK that returns to a standard BSD 3-clause license, so please use that instead when it becomes available. I'll ping you when it gets published.

Reply:
Thanks :+1:

Reply:
New SDK was released yesterday!